### DISCUSSION TITLE
SETUP: Host/Port is wrong

### DISCUSSION THREAD

**Discussion by BMiZEL** [fuatakgun/eufy_security#1255] [General]
After successful installation following steps 1) 'Eufy Security' & 2) 'WebRTC' through HACS. 

I followed instructions further under point 3) and when I reach sub-point 5 "Put Eufy Security Add-on IP Address (127.0.0.1 for Supervised installation) and configured port (default 3000) and click Submit." the installation fails.

I tried to change the IP Address to the local one of HA and also tried to use different ports - as soon as I click submit it states "Host/Port is wrong" and it resets to the default.

![Screenshot from 2024-11-23 08 41 48](https://github.com/user-attachments/assets/ae68ce30-4ee6-4962-8ed9-a9efeac17cad)

**Comment by theatricalcrab** [comment]
I am having the same issue after following the same steps. Have you had any luck so far?

**Reply by BMiZEL** [reply]
nope :-(

**Comment by johnmhedge** [comment]
me too :-(

**Reply by theatricalcrab** [reply]
Hi guys, this comment in another discussion fixed the issue for me. 

It is a step that I beleive is missed out in the instructions. I am new to Home Assistant and this step is probably second nature to most veterans. 

https://github.com/fuatakgun/eufy_security/discussions/784#discussioncomment-7302960

**Comment by johnmhedge** [comment]
"Go to Settings -> Add-ons
https://your-instance.duckdns.org/hassio/dashboard
<https://your-instance.duckdns.org/config/integrations>

Find eufy-security-ws complete the configuration then click start"
This site can’t be reached

*your-instance.duckdns.org <http://your-instance.duckdns.org>* took too
long to respond.


Perhaps someone that knows what they're talking about can help everyone
that is stalled at point 5 in the so called 'howto' to get eufy security
cameras to work in HA; please, please, please?


Thanks,

John

**Comment by IanSmallshire** [comment]
I fixed this by going to Settings -> Add-Ons ->  eufy-security-ws
Then click Configuration at the top and entering my details.
Then back to the Info Tab and select the START button to start the service.

Im pretty new to HA and still finding my way round so this may not be everything I poked at trying to make this work.

**Comment by johnmhedge** [comment]
Thanks Ian.
That was what I missed.
John
